pkg/cmd/task/task_log.go

```diff
@@ -51,14 +51,50 @@ func newCmdTaskLog(ctx api.Context) *cobra.Command {
 					}
 					return logClient.PrintTask(task.TaskID.Value, file, opts)
```

Comment:
Even though this is not part of the PR it caught my eye, how do we print multiple if we return after the first one?

Reply:
The follow case seems to be on the else statement below

Reply:
Ah, I think Robin's point is that we're on a loop here.